index.html

```diff
@@ -10,48 +10,49 @@
     <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/statement.css">
     <link rel="stylesheet" href="css/header.css">
-    <link rel="stylesheet" href="css/home.css">
+    <link rel="stylesheet" href="css/hom.css">
```

Comment:
No veo que hayas renombrado el archivo, asi que esto parece un error 
```suggestion
    <link rel="stylesheet" href="css/home.css">
```